src/openloco/input/mouse_input.cpp

```diff
@@ -1426,8 +1426,23 @@ namespace openloco::input
                 break;
             }
             case ui::scrollview::scroll_part::hscrollbar_button_right:
-                call(0x4c89ae, regs);
+            {
+                ui::window* w = WindowManager::find(_pressedWindowType, _pressedWindowNumber);
+                if (w != nullptr)
+                {
+                    w->scroll_areas[scrollAreaIndex].flags |= scroll_flags::HSCROLLBAR_RIGHT_PRESSED;
+                    int16_t widget_width = w->widgets[_pressedWidgetIndex].right - w->widgets[_pressedWidgetIndex].left - 1;
+                    if ((w->scroll_areas[scrollAreaIndex].flags & scroll_flags::VSCROLLBAR_VISIBLE) != 0)
```

Comment:
In general we dont do `!= 0` as its implicit

Reply:
I think we should write every time as this avoids confusion. Not putting the boolean expression is error prone, considered as a bad practice (from old C times) and in other languages it's even not possible to do.